metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -791,12 +925,20 @@ def _to_metadata_events(  # noqa: C901
             upstreams = []
             observed_lineage_ts = datetime.datetime.now(tz=datetime.timezone.utc)
             for view_ref in sorted(self.upstream_views):
+                # set file_path to ViewFieldType.UNKNOWN if file_path is not available to keep backward compatibility
+                # if we raise error on file_path equal to None then existing test-cases will fail as mock data doesn't have required attributes.
```

Comment:
why can't we update the test mock data?

Reply:
I am not sure whether mock data is written as per production scenario. The fields are marked as Optional in looker sdk, so to avoid in surprises in production I preferred to keep existing test as is